conda-store-server/conda_store_server/app.py

```diff
@@ -434,6 +435,7 @@ def celery_config(self):
                     "kwargs": {},
                 },
             },
+            "beat_schedule_filename": f"{CONDA_STORE_DIR}/celerybeat-schedule",
```

Comment:
This needs to use `/` and then convert to str.

Reply:
FYI, if this is related to `--beat`, that's disabled on Windows because Celery doesn't support it on Windows (I'm not clear what functionality we're losing because of that).